src/export.jl

```diff
@@ -1,231 +1,323 @@
+                        "r0" => r0,
+                        "xr" => xr,
+                        "xl" => xl,
+                        "pr" => pr,
+                        "pl" => pl,
+                        "rcut" => rcut,
+                        "maxn"=> maxn,
+                        "recursion_coefficients" => Dict("A" => [Pr.J.A[i] for i in 1:maxn],
+                                                         "B" => [Pr.J.B[i] for i in 1:maxn],
+                                                         "C" => [Pr.J.C[i] for i in 1:maxn],),
+                        "coefficients" => Dict())
+
+    for (index1, element1) in zlist_dict
+        for (index2, element2) in zlist_dict
+            pair = [reversed_species_dict[element1], reversed_species_dict[element2]]
+            ind = V2.basis.bidx0[index1, index2]
+            polypairpot["coefficients"][pair] = V2.coeffs[ind+1:ind+num_coeffs]
+        end
+    end
+    
+    return polypairpot
 end
 
 
-
-function export_ace_tests(fname::AbstractString, V, ntests = 1;
-                          nrepeat = 3, pert=0.05,
-                          s = JuLIP.chemical_symbol(i2z(V, 1)))
-   at = bulk(s, cubic=true) * nrepeat
-   JuLIP.set_pbc!(at, false)
-   r0 = JuLIP.rnn(s)
-   for n = 1:ntests
-      JuLIP.rattle!(at, pert * r0)
-      E = energy(V, at)
-      _write_test(fname * "_$n.dat", JuLIP.positions(at), E)
-   end
-   return at
+make_dimer(s1, s2, rr) = Atoms(
+    [[0.0,0.0,0.0],[rr,0.0,0.0]], 
+    [[0.0,0.0,0.0],[0.0,0.0,0.0]],
+    [atomic_mass(s1), atomic_mass(s2)],
+    [AtomicNumber(s1), AtomicNumber(s2)],
+    [100.0,100.0,100.0],
+    [false, false, false])
+
+function write_pairpot_table(fname, V2, species_dict)
+    # fname is JUST THE STEM
+    # write a pair_style table file for LAMMPS
+    # the file has a seperate section for each species pair interaction
+    # format of table pair_style is described at https://docs.lammps.org/pair_table.html
+
+    # Create filename. Only the stem is specified
+    fname = fname * "_pairpot.table"
+
+    # enumerate sections
+    species_pairs = []
+    for i in 0:length(species_dict) - 1
+        for j in i:length(species_dict) - 1
+            push!(species_pairs, (species_dict[i], species_dict[j]))
+        end
+    end
+
+    lines = Vector{String}()
+
+    # make header. date is none since ACE1 current doesnt depend on time/dates package
+    push!(lines, "# DATE: none UNITS: metal CONTRIBUTOR: ACE1.jl - https://github.com/ACEsuit/ACE1.jl")
+    push!(lines, "# ACE1 pair potential")
+    push!(lines, "")
+
+    for spec_pair in species_pairs
+        # make dimer
+        dimer = make_dimer(Symbol(spec_pair[1]), Symbol(spec_pair[2]), 1.0)
+
+        # get inner and outer cutoffs
+
+        if typeof(V2.basis.J) <: SMatrix
+            get_ru(jj) = jj.ru
+            rus = get_ru.(V2.basis.J)
+            rout = maximum(rus)    
+        else
+            rout = V2.basis.J.ru
+        end
+
+        rin = 0.1
```

Comment:
I propose reducing this to 0.001 or perhaps even smaller. Perhaps it's also good to replace with non-linear splines too at some point too...